checker.py

```diff
@@ -227,9 +225,27 @@ def git_checker(
         if not check_clean(lines):
             # WD dirty
             dirty_dirs += [gd]
-        if check_unpushed(lines):
-            # Changes unpushed
-            unpushed_dirs += [gd]
+
+    # Then, check for unpushed branches with a configured remote. This is done by using
+    # git log remote/branch..branch.
+    unpushed_branches: List[str] = []
+    for gd in itr:
+        # Retrieve all branches with a configured remote
```

Comment:
Now that this doesn't use the result of `git status`, what do you think about keeping this in its own function, and moving the subprocesses into their functions (i.e., `git status` is run in `check_clean()`, and `git config` is run in `check_unpushed()`? I worry this function is too big which hurts readability.

Reply:
With ac2b98c, I pulled the actual checks into separate functions. What do you think?

Reply:
LGTM! I like this style --- I always return results rather than mutate function args, but this actually saves lines of code, which is really nice.